UKHomeOffice/sgmr-service
# 1.2.0 SPCR-228 Refactor pagination on ManageReports

## Description
Currently, the pagination on manage voyage plans includes all plans, regardless of their status. This causes them to paginate simultaneously. E.g. the first 'page' with the first '10' results can include all 6 statuses. The desired behaviour is that the voyage plans are separated by status, with each tab (draft, submitted and cancelled) paginating separately, containing 10 results per status per page (if available). 

## To test
- Go to the manage voyage plan page
- Click through the tabs
- If there are more than 10 voyage plans on a tab, pagination should be visible
- If there are 10 or less, pagination should not be visible
- Each status/tab should paginate independently

## Developer Checklist

\* Required






Ensure you delete the branch once the Pull Request is merged.
